Zend/tests/pipe_operator/void_return.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Pipe operator fails void return chaining in strict mode
+--FILE--
+<?php
+declare(strict_types=1);
+
+function nonReturnFunction($bar): void {}
+
+try {
+    $result = "Hello World"
+        |> 'nonReturnFunction'
+        |> 'strlen';
+    var_dump($result);
+}
+catch (Throwable $e) {
+    echo $e::class, ": ", $e->getMessage(), PHP_EOL;
+}
+
+?>
+--EXPECT--
+TypeError: strlen(): Argument #1 ($string) must be of type string, null given
```

Comment:
Newline at EOL again. Please check your editor settings (and also go through the PR to check for the indicator icon or just save all the new tests once more).